Adds Bug Bounty Program documentation (#190)
pixijs/pixijs.com: docs/bug-bounty.md

# Bug Bounty Program

PixiJS is committed to delivering a reliable, high-performance rendering engine for the web. To support that mission, we’re launching a **Bug Bounty Program** to reward contributors who help make PixiJS more stable and robust.

## How It Works

1. **Bounty Tag Assignment**

>    The PixiJS team will identify eligible issues and apply the **`bounty`** label along with a specific **dollar amount**.

Reviewer: Might be helpful to link to the the open bounty issues here: 
https://github.com/pixijs/pixijs/issues?q=is%3Aissue%20state%3Aopen%20label%3A%F0%9F%92%B0Bounty

   Example: `bounty: $100`

   Only issues labeled with the `bounty` tag and a dollar amount are eligible for this program.

2. **Submission Requirements**
   To claim a bounty, you must:

   - Submit a **Pull Request (PR)** that fixes the issue.
   - Link the issue in your PR description.
   - Include a clear **example or test case** demonstrating that the bug is resolved.

>    - Follow our standard contribution guidelines.

Reviewer: ```suggestion
   - Follow our standard [contribution guidelines](https://github.com/pixijs/pixijs/blob/dev/.github/CONTRIBUTING.md).
```

3. **Approval Process**

   - The PixiJS team will review your PR.
   - If your fix is accepted and merged into the main branch, your bounty is approved.

4. **Requesting Payment**
   Once your PR is merged:

   - You can submit a payout request via our [Open Collective](https://opencollective.com/pixijs) page.
   - Include a link to the merged PR and the bounty issue in your request.
   - Payments will be processed through Open Collective.

## Terms & Conditions

- Only issues **pre-approved** with a `bounty` tag and dollar amount are eligible.
- Bounties are awarded **at the PixiJS team's discretion**. We reserve the right to reject fixes that are incomplete, introduce regressions, or do not meet project standards.
- You may submit fixes for issues without a bounty tag, but they will not be eligible for financial rewards.
- Multiple contributors can submit PRs for the same bounty, but only the PR that gets merged is eligible for payment.
- The bounty amount is fixed and non-negotiable.
- Abuse, spamming, or low-quality submissions may result in exclusion from the program.

## **Finding Bounty Issues**

> You can easily find eligible bounty issues by filtering them on our GitHub repository:
> 
> ![Bounty Issues](/images/misc/bug-bounty-label.png)
> 
> - Go to the [PixiJS Issues](https://github.com/pixijs/pixijs/issues) page.
> - Click the **Labels** dropdown menu at the top of the issues list (as shown below).
> - Select the `bounty` label to view only bounty-tagged issues.
> 
> This allows you to focus your contributions on tasks that have a financial reward.

Reviewer: ```suggestion
You can easily find [eligible bounty issues](https://github.com/pixijs/pixijs/issues?q=is%3Aissue%20state%3Aopen%20label%3A%F0%9F%92%B0Bounty) by on our GitHub repository.

This allows you to focus your contributions on tasks that have a financial reward.
```

## Why We're Doing This

We believe in the power of open source and community collaboration. Our bug bounty program is designed to:

- Encourage contributors to tackle important, impactful issues.
- Recognize the hard work involved in debugging and fixing complex problems.

> - Make PixiJS even better for everyone.

Reviewer: Another reason, to be transparent, is that the core team is often tackling other important things and cannot get to all issues quickly.

- The bounty amount is fixed and non-negotiable.
- Abuse, spamming, or low-quality submissions may result in exclusion from the program.

> Here’s a polished version of that section with consistent tone, formatting, and improved clarity:

Reviewer: ```suggestion
```
